index.js

```diff
@@ -3,71 +3,119 @@ var userName = prompt('Hi! What\'s your name?');
+// } else if (questionTwo === 'no' || questionTwo === 'no') {
+//   alert ('I know, I probably fooled you when I talked about ping pong in my intro. That\'s a close runner up!');
+// } else {
+//   var altTwo = prompt ('Sorry, no shades of gray allowed. Yes or no only, please.');
+//   altTwo = altTwo.toLowerCase();
+//   if (altTwo === 'yes' || altTwo === 'y') {
+//     alert('You got it! I still try to play twice a week in an adult leage I\'ve been a part of since I moved to Seattle!');
+//   } else if (altTwo === 'no' || altTwo === 'n'){
+//     alert('It get\'s a lot of criticism for being boring because so few goals are scored but every second is exciting if you know what to watch for!');
+//   } else {
+//     alert('Yes or no. It\'s so simple!');
+//   }
+// };
+//
+// var questionThree = prompt ('Do I live in Fremont?', 'Yes or no...');
+// questionThree = questionThree.toLowerCase();
+// console.log(questionThree);
+// if (questionThree === 'yes' || questionThree === 'y') {
+//   alert ('Yep. Got a nice little one bedroom condo right there in the Center of the Universe.');
+// } else if (questionThree === 'no' || questionThree === 'no') {
+//   alert ('Can\'t imagine living anywhere else!');
+// } else {
+//   alert ('Still don\'t have it down by now, huh? A simple yes or no will do!');
+// };
+//
+// var questionFour = prompt ('Do I play a musical instrument?', 'Yes or no...');
+// questionFour = questionFour.toLowerCase();
+// console.log(questionFour);
+// if (questionFour === 'yes' || questionFour === 'y') {
+//   alert ('I learned how to play guitar in high school and still make noise with it now and again.');
+// } else if (questionFour === 'no' || questionFour === 'no') {
+//   alert ('OK...just because I\'m not very good at it doesn\'t mean that I don\'t \'play\' it!');
+// } else {
+//   alert ('Real slow on the uptake, eh? Please, for the love of god, just a yes or a no.');
+// };
+//
+var questions = [
+  'Was I born in Pennsylvania?',
+  'Is soccer my favorite sport?',
+  'Do I live in Fremont?',
+  'Do I play a musical instrument?',
+  'Do I love to drive?'
+];
+var numberOfQuestions = questions.length;
+var answers = [
+  'yes',
+  'yes',
+  'yes',
+  'yes',
+  'no'
+];
+var correctFeedback = [
+  'You\'re right! I was born just outside of Philadelphia.',
+  'Oh yeah! I still play quite a bit and go to all the Sounders games.',
+  'Right! The center of the universe! I love my neighborhood.',
+  'Correct! I play the guitar and the trumpet.',
+  'You got it! I\'d much rather ride my bicycle!'
+];
+var incorrectFeedback = [
+  'Sorry. I know I don\'t have the accent but I was born near Philly.',
+  'Wrong. I know, I said that I really liked ping pong but not as much as the beautiful game!',
+  'You guessed incorrectly. I couldn\'t imagine living anywhere else!',
+  'Wrong. I do play the guitar. Not very well but...',
+  'Nope. It is actually one of my least favorite things to do.'
+];
+
+var correctResponses = [];
```

Comment:
Another way to accomplish this, rather than pushing to an array and checking the length, is to just make this an integer and increment by one each time the user gets a question right.